src/create-droppable.ts

```diff
@@ -0,0 +1,77 @@
+import { useDragDropContext } from "./drag-drop-context";
+import {
+  elementLayout,
+  noopTransform,
+  Transform,
+  transformsAreEqual,
+} from "./layout";
+import { transformStyle } from "./style";
+import {
+  createRenderEffect,
+  createSignal,
+  onCleanup,
+  onMount,
+  Setter,
+} from "solid-js";
+
+interface Droppable {
+  (element: HTMLElement): void;
+  ref: Setter<HTMLElement | null>;
+  get isActiveDroppable(): boolean;
+}
+
+const createDroppable = (
+  id: string | number,
```

Comment:
Question: Now that ids are consistently typed as `string | number` throughout the code, would it make sense to have this as a dedicated type alias or not? E.g. `type ID = number | string;`?

Reply:
Though about it. However, it's a relatively simple type that's unlikely to change, so importing it everywhere (presumably from a dedicated *types.ts* file or something) feels unnecessary.